specification/dartLangSpec.tex

```diff
@@ -10552,8 +10974,10 @@ \subsection{Throw}
 the stack trace $t$ is stored on $v$ so that it will be returned
 by the $v$'s \code{stackTrace} getter
```

Comment:
(Maybe say: `by the \code{stackTrace} getter inherited from \code{Error}.`, because it might be overridden in a subclass, so it's not the `stackTrace` getter of *v*).

Reply:
Good catch, done!